backend/tracim_backend/lib/webdav/resources.py

```diff
@@ -400,9 +400,19 @@ def moveRecursive(self, destpath):
                 raise DAVError(HTTP_FORBIDDEN)
 
             try:
-                self.workspace.label = webdav_convert_file_name_to_bdd(basename(normpath(destpath)))
+                workspace_api = WorkspaceApi(
+                    current_user=self.user,
+                    session=self.session,
+                    config=self.provider.app_config,
+                )
+                workspace_api.update_workspace(
+                    workspace=self.workspace,
+                    label=webdav_convert_file_name_to_bdd(basename(normpath(destpath))),
+                    description=self.workspace.description
+                )
                 self.session.add(self.workspace)
                 self.session.flush()
+                workspace_api.execute_update_workspace_actions(self.workspace)
```

Comment:
remove this: it must be implicit from outside of workspace_api

Reply:
see : https://github.com/tracim/tracim/pull/1578#discussion_r274287675

Reply:
https://github.com/tracim/tracim/issues/1589